private-gke-cluster/outputs.tf

```diff
@@ -12,3 +12,8 @@ output "gke_cluster_name" {
 output "gke_control_plane_cidr" {
   value = google_container_cluster.gke_cluster.private_cluster_config[0].master_ipv4_cidr_block
 }
+
+# ipv4 cidr block for pods running in the cluster
+output "gke_pods_ipv4_cidr_block" {
```

Comment:
Just for my own knowledge, are there additional cidr blocks beyond the control plane block and the cluster block or just those two?

Reply:
The networks that we typically worry about in GKE deployments are:

- The master (control plane) CIDR range: This is the range that IP addresses for the kubernetes API are assigned from. It's not required to set this, but we explicitly define it in gnomAD, because we need to create firewall rules to allow the master nodes to perform webhooks on the elasticsearch services running in the cluster.
- The VPC subnet that the GKE node pools/nodes are deployed in: This is created with our VPC module, and then passed in as the `subnet` in the `google_container_cluster` resource
- The pods CIDR range (sometimes also called the "cluster" ipv4 CIDR range): This is the network range where IP addresses for individual pods are assigned from.
- The services CIDR range: This is the range that Service IP addresses are assigned from, when you create a kubernetes [Service](https://kubernetes.io/docs/concepts/services-networking/service/) objects.